src/login/LoginMain/LoginMain.js

```diff
@@ -45,6 +47,11 @@ class Main extends Component {
                   path="/"
                   render={props => <div {...props} ></div>}
                 />
+                <Route
+                  exact
+                  path="/reset/reset-password/"
+                  render={props => <InitResetFormContainer {...props} />}
```

Comment:
I think we could call this component `<ResetPassword />` and let it provide the structure for all of the elements that enable the reset password process

Reply:
In the subsequent steps resetting the password, after receiving an email and clicking on the link, this component should not need to participate at all, so I'm not sure.

This component holds the form that initiates the reset password flow, that's why I called it InitResetForm :)